src/utils/database.ts

```diff
@@ -499,9 +495,7 @@ export class Transcript {
                     out += `> [Crosspost From] ${message.referencedMessage[0]} in ${message.referencedMessage[1]} in ${message.referencedMessage[2]}\n`;
                 } else out += `> [Reply To] ${message.referencedMessage}\n`;
             }
-            out += `${message.author.nickname ?? message.author.username}#${message.author.discriminator} (${
-                message.author.id
-            }) (${message.id})`;
+            out += `${message.author.nickname ?? message.author.username} (${message.author.id}) (${message.id})`;
```

Comment:
Many people still have discriminators, should we be removing them from our code yet? Better may be to check if a discriminator exists (by checking if it's 0) and then conditionally showing it